component/common/api/wifi/wifi_conf.c

```diff
@@ -3787,7 +3787,7 @@ int wifi_get_antenna_info(unsigned char *antenna)
 	rtw_memset(buf, 0, sizeof(buf));
 	rtw_memcpy(buf, "get_ant_info", 12);
 	ret = wext_private_command_with_retval(WLAN0_NAME, buf, buf, 32);
-	sscanf(buf, "%d", antenna); // 0: main, 1: aux
+	sscanf(buf, "%c", antenna); // 0: main, 1: aux

```

Comment:
The antenna value would never be negative, would it?

Reply:
looks like its only ever "0" or "1"